oci/layout/oci_dest.go

```diff
@@ -127,13 +128,13 @@ func createManifest(m []byte) ([]byte, string, error) {
 		}
 		return b, om.MediaType, nil
 	case manifest.DockerV2ListMediaType:
-		return nil, "", fmt.Errorf("can't create OCI manifest from Docker V2 schema 2 manifest list")
+		return nil, "", errors.New("can't create an OCI manifest from Docker V2 schema 2 manifest list")
 	case imgspecv1.MediaTypeImageManifestList:
-		return nil, "", fmt.Errorf("can't create OCI manifest from OCI manifest list")
+		return nil, "", errors.New("can't create an OCI manifest from OCI manifest list")
 	case imgspecv1.MediaTypeImageManifest:
 		return m, mt, nil
 	}
-	return nil, "", fmt.Errorf("Unrecognized manifest media type: %q", mt)
+	return nil, "", errors.New("unrecognized manifest media type")
```

Comment:
We should still be quoting the type here; it has helped us once, it may help again.